src/lib/pages/backoffice/Actions/CheckOut/CheckOutResults/CheckOutResults.js

```diff
@@ -0,0 +1,49 @@
+import { Loader } from '@components/Loader';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Header, Segment } from 'semantic-ui-react';
+import { ItemList } from './ItemList';
+import { PatronList } from './PatronList';
+
+export default class CheckOutResults extends Component {
+  componentDidMount() {
+    const { clearResults } = this.props;
+    clearResults();
+  }
+
+  render() {
+    const { itemList, isLoading, patronList } = this.props;
+
+    // NOTE: if we have 1 result we will get redirected, avoid render list with 1
+    const hasItems = itemList.length > 1;
+    const hasPatrons = patronList.length > 1;
+    const hasResults = hasItems || hasPatrons;
+
+    return (
+      <Loader isLoading={isLoading}>
+        {hasPatrons && <PatronList patrons={patronList} />}
+        {hasItems && <ItemList items={itemList} />}
+        {!hasResults && (
+          <Segment placeholder>
+            <Header textAlign="center">
+              Scan a Patron badge OR a physical copy barcode
```

Comment:
Same as before, you are not use how this will be used, maybe there is no scan device:
```suggestion
              Insert patron id/email or physical copy barcode
```